test/qtiCommonRenderer/interactions/graphicOrder/test.js

```diff
@@ -3,7 +3,7 @@ define([
     'taoQtiItem/runner/qtiItemRunner',
     'core/mouseEvent',
     'ui/interactUtils',
-    'json!taoQtiItem/test/qtiCommonRenderer/interactions/GraphicOrder/sample.json'
+    'json!taoQtiItem/test/qtiCommonRenderer/interactions/graphicOrder/sample.json'
```

Comment:
**praise:** Nice catch!

Reply:
I tried to reproduce it this way:
```
git checkout develop
rm -rf ./node_modules
npm ci
...
git checkout chore/tr-3008/upgrade-3rd-party-deps
npm ci
...
```
And it worked for me. I had the issue you are describing, but it was fixed in the last commit